storagenode/storagenodedb/versions.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+// versions represents the database that contains the database schema version history.
+type versions struct {
+	location string
+	SQLDB
+}
+
+func newVersions(db SQLDB, location string) *versions {
+	return &versions{
+		location: location,
+		SQLDB:    db,
+	}
+}
+
+// Rebind rebind parameters
+func (db *versions) Rebind(s string) string { return s }
```

Comment:
What is the purpose here? Above you mentioned how `Rebind` was a nop, but it looked like you removed it? Will this be implemented later?

Either way, if we keep this, I think a better comment explaining what we hope to do here (at least in the future) would be worthwhile.

Reply:
It looks like this was a left over API call satisfying an interface function for no reason and I couldn't find any code that used it for any purpose so I didn't see the need to keep dead code around.

Reply:
Done!